### PR TITLE
Patch RT1060x registers

### LINKED ISSUE #36: Missing register field - FlexSPI - `MCR0` - `SERCLKDIV`
Various MCUs are missing the `SERCLKDIV` field in the FlexSPI `MCR0` register. These fields are missing in the SVD, so they're also missing in this crate. To fix this, use SVD patches to re-introduce the field.

This affects the following families and MCUs.

- 1020
  - 1021
- 1050
  - 1051
  - 1052
- 1060
  - 1061
  - 1062
  - 1064

The 1011 and 1176 SVDs are not affected. This field is marked as "reserved" in an older 1015 reference manual. Download the latest RM and double check this once NXP's website exits maintenance mode...

### PR DESCRIPTION
**Pull request by Akari202**
I tried to fix #36 and and I added the missing FILT_PRSC that i encountered. I did'nt know if these patches also should apply to the other chips so i only added them to the rt106x chips. 

I have never done this kinda thing before so feel free to let me know what i messed up but i think everything should be correct.

### CODE REVIEW

File: devices/common_patches/flexspi.yaml [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# Add SERCLKDIV to FLEXSPIx MCR0 registers 
+FLEXSPI*: 
+  MCR0: 
+    _add: 
+      SERCLKDIV: 
+        description: serial root clock divisor
+        bitOffset: 8
+        bitWidth: 10
```
Comment (mciantyre) [inline devices/common_patches/flexspi.yaml]: The bitwidth is 3 bits. Confirmed in the 1060 and 1020 RMs.
```suggestion
        bitWidth: 3
```

**Review by mciantyre — CHANGES_REQUESTED** [review]
Thanks for jumping in. Looks like a good start, and we can build on it with some small patches.

- I updated #36 to describe all MCUs missing the `SERCLKDIV` field. There's more than just the 1060, and it would be helpful to fix all affected parts in one go.
- Based on the reference manuals I have available, the ENC change also affects the 1021.
- Check in the generated Rust code. Without this, a CI check fails. See the contributing doc to learn about the workflow.

**Comment by Finomnis** [comment]
Minor nit: don't check in files without newline termination; most IDEs have an option to correct this automatically.

In VSCode, for example, the relevant settings are:
```
"files.trimTrailingWhitespace": true,
"files.trimFinalNewlines": true,
"files.insertFinalNewline": true,
```

**Comment by Akari202** [comment]
> The 1011 and 1176 SVDs are not affected. This field is marked as "reserved" in an older 1015 reference manual. Download the latest RM and double check this once NXP's website exits maintenance mode...

Checking the reference manuals this change does effect the 1015. Now that NXP's website is finally back i have checked these registers for all the chips supported by this ral and everything should be properly applied now. I also checked in the generated code so the CI checks should be able to run properly this time

**Review by mciantyre — APPROVED** [review]
Looks great. Thanks for making these changes throughout all supported chips. I'll release this in the 0.5 series.

**Pull request merged**